src/gl/pointFeature.js

```diff
@@ -261,10 +261,19 @@ geo.gl.pointFeature = function (arg) {
       position[i3] = posVal.x;
       position[i3 + 1] = posVal.y;
       position[i3 + 2] = posVal.z || 0;
+      nonzeroZ = nonzeroZ || position[i3 + 2];
     }
     position = geo.transform.transformCoordinates(
                   m_this.gcs(), m_this.layer().map().gcs(),
                   position, 3);
+    /* Some transforms modify the z-coordinate.  If we started with all zero z
+     * coordinates, don't modify them.  This could be changed if the
+     * z-coordinate space of the gl cube is scaled appropriately. */
```

Comment:
Yes, proj4 sometimes produces non-zero z-coordinates.  This happens, for instance, in some of the projections using datums other than WGS84/NAD83, where there is a different ellipsoid.